core/src/main/java/bisq/core/api/CoreOffersService.java

```diff
@@ -67,6 +68,16 @@ public CoreOffersService(CreateOfferService createOfferService,
         this.user = user;
     }
 
+    Offer getOffer(String id) {
+        List<Offer> offers = offerBookService.getOffers().stream()
+                .filter(o -> o.getId().equals(id))
+                .collect(Collectors.toList());
```

Comment:
Not wrong, but it's a bit odd to make list when we're looking for one element. `.findAny()` returns an `Optional<>` which you could then do `offer.isPresent()` and `return offer.get()` making it clear we only expect to find one offer.

Reply:
Resolved by commit 8394bd1